content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -67,7 +67,7 @@ See the [webhook request](#request) section for details on the data sent to webh
 See the [webhook response](#response) section for the data expected from webhooks.
 
 The example admission webhook server leaves the `ClientAuth` field
-[empty](https://github.com/kubernetes/kubernetes/blob/v1.13.0/test/images/webhook/config.go#L47-L48),
+[empty](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/test/images/agnhost/webhook/config.go#L38-L39),
```

Comment:
If hyperlinking to a specific line, it's better to pin to a named branch (the latest release is a good choice). Otherwise if that file changes the line might end up reading something else.

Reply:
I have check these specific line in version 1.17 to 1.20 it points to same items. 😊 If we change source code, I think it is a good idea to change website too.

Reply:
Hi @Abirdcfly .
It would be helpful to get this link updated.
Since the parameter `githubbranch` resolves to `master`, would you change this section of the link
to the current version, `1.20.0`, or possibly the variable, `fullversion`?
This link may change in the future?